app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt

```diff
@@ -132,14 +135,42 @@ class OngoingTopicListActivityTest {
     }
   }
 
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
+    launch<OngoingTopicListActivity>(createOngoingTopicListActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )
+      onView(atPositionOnView(R.id.ongoing_topic_list, 1, R.id.topic_name_text_view)).check(
+        matches(
+          withText(containsString("Ratios and Proportional Reasoning"))
+        )
+      )
+      onView(isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          0
+        )
+      )
+      onView(atPositionOnView(R.id.ongoing_topic_list, 1, R.id.topic_name_text_view)).check(
```

Comment:
Test case is failing because you index is incorrect. Instead of `1` it should be 0.

Reply:
Changed